client/app/styles/base/_normalize.styl

```diff
@@ -0,0 +1,9 @@
+html
+body
+    margin: 0
+    padding: 0
```

Comment:
I'm not sure those are really needed as we already embed `normalize.css` which already sets `margin` for `body`